datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/JavaTimeSerializerModifier.java

```diff
@@ -0,0 +1,32 @@
+package com.fasterxml.jackson.datatype.jsr310.ser;
+
+import java.time.Month;
+import java.util.function.Supplier;
+
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializationConfig;
+import com.fasterxml.jackson.databind.ser.BeanSerializerModifier;
+
+/**
+ * @since 2.17
+ */
+public class JavaTimeSerializerModifier extends BeanSerializerModifier {
+    private final boolean _oneBaseMonths;
+    private final Supplier<Boolean> _serializeEnumsByIndex;
+
+    public JavaTimeSerializerModifier(boolean oneBaseMonths, Supplier<Boolean> serializeEnumsByIndex) {
```

Comment:
As per other discussions, can drop `Supplier` and checks altogher as Serializer is only used if one-based-months feature is set.